runtimes/cpp-17/src/RuntimeResponse.h

```diff
@@ -0,0 +1,47 @@
+#ifndef CPP_RUNTIME_RUNTIMERESPONSE_H
+#define CPP_RUNTIME_RUNTIMERESPONSE_H
+
+#include <string>
+#include <json/value.h>
+#include "RuntimeRequest.h"
+
+namespace runtime
+{
+    struct RuntimeResponse
+    {
+        std::string data;
+        int statusCode{200};
+
+        RuntimeResponse &json(const Json::Value &json, int code = 200)
+        {
+            data = json.toStyledString();
+            statusCode = code;
+            return *this;
+        }
+
+        RuntimeResponse &send(const std::string &string, int code = 200)
+        {
+            data = string;
+            statusCode = code;
+            return *this;
+        }
+
+        static RuntimeResponse error(const std::exception &e)
+        {
+            RuntimeResponse resp;
+            resp.data = e.what();
+            resp.statusCode = 500;
+            return resp;
+        }
+
+        static RuntimeResponse unauthorized()
+        {
+            RuntimeResponse resp;
+            resp.data = R"({"code": 401, "message": "Unauthorized"})";
+            resp.statusCode = 500;
```

Comment:
We no longer send JSON strings as error messages 

https://github.com/open-runtimes/open-runtimes/blob/3261573d3a02e7dbad1868e465d2858a3a183494/runtimes/deno-1.13/server.ts#L13